readme.md

```diff
@@ -0,0 +1,26 @@
+# Laravel Media plugin
+
+## Introduction
+
+This plugin adds the ability for users to work with media files.
+
+## Installation
+
+1. Install the package using the following command: `composer require ronasit/laravel-media`
+2. Run `php artisan vendor:publish`
+3. Add `src/MediaServiceProvider` to `config/app.php` your application
+
+### Configs
+
+- `media.permitted_types` - list of allowed media file types.
+- `media.classes.user_model` - user model class that will have a relationship with the media class.
```

Comment:
Let's remove configs block from the readme and move user configuration step to the installation block

Reply:
Done.